combat-trainer.lic

```diff
@@ -566,7 +566,11 @@ class SpellProcess
     echo("  @offensive_spell_cycle: #{@offensive_spell_cycle}") if $debug_mode_ct
 
     @spell_data = get_data('spells').spell_data
-
+    @offensive_spells.each.with_index do |spell, i| 
+      if @spell_data[spell['name']]
+        @offensive_spells[i] = spell.merge(@spell_data[spell['name']]) 
```

Comment:
After some testing, it still needs to be there to grab the custom `prep` if there is one. I've added it back